components/stash/stash.html

```diff
@@ -10,7 +10,7 @@ <h4 class="stash-toggle-text" data-bind="click: toggleShowStash">
     </h4>
     <div class="list-group" data-bind="foreach: stashedChanges">
       <div class="list-group-item">
-        <a href="#" class="stash-apply octicon-circled pull-left bootstrap-tooltip" data-bind="html: applyIcon, click: apply" data-toggle="tooltip" title="Apply this stash"></a>
```

Comment:
The clicktest did fail on macOS because this used floats and could be clicked. It now works by using `display: inline-block` which is nicer.